tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -97,9 +97,9 @@ EventsAndTriggers:
           Offset: 0
     - - PsiAlongAxis1
       - PsiAlongAxis2
-  - - Times:
-        Specified:
-          Values: &FinalTime [2000.]
+  - - TimeCompares:
+        Comparison: GreaterThan
+        Value: 2000.
```

Comment:
I am concerned if this didn't work, and I am even more concerned if the new version does.  (For the latter, because you removed the definition of `FinalTime`.)

What was the reason for this change?

Reply:
I put the removal of the `StepToTimes` in the wrong commit  but fixed it now. 

This always worked, the reason is just to simplify things. The system reaches a steady-state solution so we do not care about a point landing, it just needs to roughly complete at some time.